src/components/tiles/table/cell-text-editor.tsx

```diff
@@ -71,40 +84,39 @@ export default function CellTextEditor<TRow, TSummaryRow = unknown>({
 
   return (
     <Portal>
-      <TextareaAutosize
-        value={value}
-        className={`rdg-text-editor ${RDG_INTERNAL_TEXT_EDITOR_CLASS} ${linked && 'linked'}`}
-        style={{top, left, width: column.width}}
-        autoFocus={true}
-        onChange={event => {
-          updateValue(event.target.value);
-        }}
-        onFocus={event => {
-          // Select all text when focused, but not until after the current event
-          // has been processed. Otherwise, starting to edit a cell with a
-          // keystroke will select the text and then overwrite it all immediately.
-          setTimeout(() => {
+      <div className={`rdg-editor-container ${RDG_INTERNAL_EDITOR_CONTAINER_CLASS}`}>
+        <TextareaAutosize
+          value={value}
+          className={`rdg-text-editor ${RDG_INTERNAL_TEXT_EDITOR_CLASS} ${linked && 'linked'}`}
+          style={{
+            background: "white",
+            display: "block",
+            left,
+            position: "absolute",
+            top,
+            width: column.width
+          }}
```

Comment:
A comment about the different behavior on production & development would be important here, so that the next developer doesn't remove these without testing on the server.